stories/Chessboard.stories.tsx

```diff
@@ -1084,3 +1090,170 @@ export const BoardWithCustomArrows = () => {
     </div>
   );
 };
+
+///////////////////////////////////
+////////// ManualBoardEditor //////
+///////////////////////////////////
+export const ManualBoardEditor = () => {
+  const game = useMemo(() => new Chess("8/8/8/8/8/8/8/8 w - - 0 1"), []); // empty board
+  const [boardOrientation, setBoardOrientation] =
+    useState<"white" | "black">("white");
+  const [boardWidth, setBoardWidth] = useState(360);
+  const [fenPosition, setFenPosition] = useState(game.fen());
+
+  const handleSparePieceDrop = (piece, targetSquare) => {
+    const color = piece[0];
+    const type = piece[1].toLowerCase();
+
+    const success = game.put({ type, color }, targetSquare);
+
+    if (success) {
+      setFenPosition(game.fen());
+    } else {
+      alert(
+        `The board already contains ${color === "w" ? "WHITE" : "BLACK"} KING`
+      );
+    }
+
+    return success;
+  };
+
+  const handlePieceDrop = (sourceSquare, targetSquare, piece) => {
+    const color = piece[0];
+    const type = piece[1].toLowerCase();
+
+    // this is hack to avoid chess.js bug, which I've fixed in the latest version https://github.com/jhlywa/chess.js/pull/426
+    game.remove(sourceSquare);
+    game.remove(targetSquare);
```

Comment:
can this be updated now?

Reply:
In order to update it now, we have to upgrade our `chess.js` package version

Reply:
okay, that can be tackled separately